### DISCUSSION TITLE
Union type in the schema

### DISCUSSION THREAD

**Discussion by SomiDivian** [prisma/prisma#15049] [Q&A]
Why is it not available to set a field type to a union type?
like in Apollo Graphql https://www.apollographql.com/docs/apollo-server/schema/unions-interfaces/

**Comment by nurul3101** [comment]
Hi @SomiDivian 👋 
We have a feature request for adding support for union types: [#2505](https://github.com/prisma/prisma/issues/2505)
If you could leave a 👍 to the feature request then it would help our product team in prioritising this.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
